pipelines/release/weekly_release.groovy

```diff
@@ -176,13 +176,25 @@ notify.wrap {
       //  stages within other parallel stages, so we do these all
       //  sequentially in order not to have them held up by validate_drp.
 
-      stage('SAL-sciplat-lab linked image set (master/summit/base/kueyen/nts/tts') {
+      stage('SAL-sciplat-lab linked image set (master/summit/base/kueyen/tts') {
```

Comment:
Didn't notice the unclosed paren before, but might be nice to fix it now.

Reply:
TY

Reply:
Done